lib/Constants.js

```diff
@@ -268,6 +268,10 @@ Constants.Errors = {
     NAME: 'ECONNRESET',
     MESSAGE: 'A connection was forcibly closed by a peer.'
   },
+  DeploymentVersionTooLow: {
+    NAME: 'DeploymentVersionTooLow',
+    MESSAGE: 'The specified service version is lower than the one deployed last. Please bump the version in package.json and try again.'
```

Comment:
The message is incorrect:
- The version is not lower, it could be either lower or equal
- The version is not "specified" but is taken from the package.json
I would phrase it like this:
'Deployment failed: local version ({local_version}) is lower or equal to the cloud version ({cloud_version}).'